assets/src/components/reader-theme-carousel/index.js

```diff
@@ -149,22 +149,23 @@ export function ReaderThemeCarousel() {
 		<div className="reader-theme-selection">
 			<p>
 				{
-					// @todo Probably improve this text.
-					__( 'Select the theme template for mobile visitors', 'amp' )
+					__( 'Choose the theme to be used for AMP pages. This theme will normally be exclusively shown to mobile visitors when using Reader template mode.', 'amp' )
```

Comment:
@amedina FYI: This string change.

![image](https://user-images.githubusercontent.com/134745/109256808-a669da00-77ab-11eb-9b34-89d00dbe057a.png)